### PR TITLE
Rundeck v2.9.3, bintray https & apt repo

### PR DESCRIPTION
**Pull request by gsreynolds**
### Description

* Bump default Rundeck version installed to latest, v2.9.3
* Use https for bintray and use the bintray apt repos for rundeck - workaround two apt signing key issue

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable

---
This change is [Reviewable](https://reviewable.io/reviews/sous-chefs/rundeck/178)

### CODE REVIEW

**Comment by nilroy** [comment]
When are we getting this PR merged?

File: recipes/server_install.rb [hunk 1]
```diff
@@ -51,23 +51,35 @@
...
+
+  # Hack to get multiple keys installed without borrowing apt_repository internals
+  # Abuse name property so we only write out one file in sources.list.d/
+  apt_repository 'rundeck-build' do
+    name 'rundeck'
+    uri node['rundeck']['deb']['repo']['url']
+    distribution '/'
+    key node['rundeck']['deb']['repo']['gpgkey']
+    action :add
   end
```
Comment (gsreynolds) [inline recipes/server_install.rb]: This issue that requires this workaround is fixed in Chef 13.4.10, which supports an array of repo signing keys. We can possibly add a version check and if chef version `>= 13.4.10`, do the following:
```
apt_repository 'rundeck' do
   mode '0644' 
   uri node['rundeck']['deb']['repo']['url']
   distribution '/'
   key [node['rundeck']['deb']['repo']['key'], node['rundeck']['deb']['repo']['gpgkey']]
   action :add
 end
```
else do what is there just now.

13.4 isn't stable yet and we need to support 12 and earlier versions of 13.

**Comment by ev0ldave** [comment]
Reviewed 5 of 5 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/sous-chefs/rundeck/178)*

**Comment by nilroy** [comment]
when can we expect to bump the version of this cookbook in the supermarket?

**Pull request merged**